Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.18 as builder
```

Comment:
Should we use `1.19`?

Reply:
not sure, I mean is the most recent version of go, but I think the sdk works with 1.18

Reply:
make sense, bumbed the version.